pkg/controller/master/setting/vip_pools_config.go

```diff
@@ -58,11 +64,29 @@ func (h *Handler) syncVipPoolsConfig(setting *harvesterv1.Setting) error {
 }
 
 func ValidateCIDRs(pools map[string]string) error {
-	for ns, v := range pools {
-		cidrs := strings.Split(v, ",")
-		for _, cidr := range cidrs {
-			if _, _, err := net.ParseCIDR(cidr); err != nil {
-				return fmt.Errorf("invalid CIDR value %s of %s, error: %s", v, ns, err.Error())
+	for ns, pool := range pools {
```

Comment:
can we move the validation to webhook instead?

Reply:
This function is only called by webhook